src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -63,169 +64,194 @@ field. If a new model is used, common sense should be used to derive a name
-| ------------ | ------------------------------------------------------------------------------------------------------------------------------------ |
-| Model        | Model label - see table above                                                                                                        |
-| Shells       | OPTIONAL. Shells that were utilized to fit the model, as a list of b-values. If the key is not present, all shells were used.        |
-| Gradients    | OPTIONAL. Subset of gradients utilized to fit the model, as a list of three-elements lists. If not present, all gradients were used. | ModelDescription | OPTIONAL. Extended information to describe the model. |
-| ModelURL     | OPTIONAL. URL to the implementation of the specific model utilized.                                                                  |
-| Parameters   | OPTIONAL. A dictionary of model parameters (see below for keys specific to each model)                                               |
+| **Key name**     | **Description**                                                                                                                      |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
+| Shells           | OPTIONAL. Shells that were utilized to fit the model, as a list of b-values. If the key is not present, all shells were used.        |
+| Gradients        | OPTIONAL. Subset of gradients utilized to fit the model, as a list of three-elements lists. If not present, all gradients were used. |
+| ModelDescription | OPTIONAL. Extended information to describe the model.                                                                                |
+| ModelURL         | OPTIONAL. URL to the implementation of the specific model utilized.                                                                  |
+| Parameters       | OPTIONAL. A dictionary of model parameters (see below).                                                                              |
 
--   Model:
+Parameters that may be stored within the JSON sidecar "Parameters" field depend on the particular model used:
 
-    -   `DTI` :
+-   `DTI` :
 
-        -   `FitMethod` : {`WLS`,`OLS`,`NLLS`,`RESTORE`}
-        -   `RESTORE` : sigma
+    -   `FitMethod` : {`WLS`,`IWLS`,`OLS`,`NLLS`}
+    -   `OutlierRejection`: boolean
+    -   `RESTORESigma` : value
 
-    -   `DKI` and `WMTI`:
+-   `DKI` and `WMTI`:
 
-        -   `FitMethod` : {`WLS`,`OLS`,`NLLS`}
+    -   `FitMethod` : {`WLS`,`IWLS`,`OLS`,`NLLS`}
 
-    -   `CSD`:
+-   `CSD`:
 
-        -   `SphericalHarmonicOrder` : value
-        -   `ResponseFunctionOrder` : value
-        -   `ResponseFunction` : \[1:response_function_order_value,b0mean\]
-        -   `Basis` : {`MRtrix 0.2`,`MRtrix3`,`DESCOTEAUX`}
+    -   `Tissue` : string

```

Comment:
Should this be a controlled vocabulary, listing the possible tissue types?

Reply:
hi @jchoude thanks, how would you change this specifically?

Reply:
Just wondering if we should have a list of possible tissues and the spelling, like `[white|gray|csf]`, to avoid people writing the same tissue type in various ways , eg: `'white matter', 'WM', 'white'`, etc.

Reply:
I'd be open to a *limited* controlled vocabulary; i.e. have pre-defined strings recommended for use, but still permit use of items not in the vocabulary. While MSMT is most commonly done with WM / GM / CSF tissues, it's in no way obliged to use such, and in time there will be an increasing number of use cases that don't conform to such. Is there an appropriate syntax for such in the spec?

Also note that I would generally expect an abbreviated name of the tissue to appear in the file name, probably under the "`desc-`" tag. So the "`Tissue`" string in the JSON is in one way redundant; but conversely it does give the opportunity for a more verbose name than what would be crammed into the filename.

Reply:
Yes, rethinking about it we might not want to add another vocabulary at this point. Let's leave it like that.